core/amber/src/test/scala/edu/uci/ics/amber/engine/e2e/Utils.scala

```diff
@@ -29,7 +29,7 @@ object Utils {
       WorkflowInfo(operators, links, mutable.MutableList[BreakpointInfo]()),
       context
     )
-    texeraWorkflowCompiler.init()
+
```

Comment:
i am forgetting. why are we deleting this?

Reply:
WorkflowCompiler now invokes `init()` in its constructor. I am removing this here as it is redundant now.